unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/Analytics/IAnalytics.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using DCL.Interface;
+
+public interface IAnalytics : IDisposable
+{
+    void SendAnalytic(string eventName, Dictionary<object, object> data);
+}
+
+public class Analytics : IAnalytics
+{
+    //Remove this once environment is on its own assembly and can be accessed properly
+    public static IAnalytics i;
+    public Analytics() { i = this; }
+
+    public void SendAnalytic(string eventName, Dictionary<object, object> data) { SendToSegment(eventName, data); }
+
+    internal void SendToSegment(string eventName, Dictionary<object, object> data) { WebInterface.SendGenericAnalytic(eventName, data); }
```

Comment:
Could we add a `VERBOSE` flag to be able to look at console's log what analytics are being send?

Reply:
That's a great idea, thanks!